migrations/1597654308406-remove-importHashes-from-users-with-accounts.js

```diff
@@ -0,0 +1,61 @@
+const { info, error } = require('../src/logger');
+
+const { connect, close } = require('../src/utils/database');
+const { userModel } = require('../src/services/user/model/index');
+const accountModel = require('../src/services/account/model');
+
+// How to use more than one schema per collection on mongodb
+// https://stackoverflow.com/questions/14453864/use-more-than-one-schema-per-collection-on-mongodb
+
+
+// TODO npm run migration-persist and remove this line
+// TODO update seed data and remove this line
+
+module.exports = {
+	up: async function up() {
+		await connect();
+		// ////////////////////////////////////////////////////
+		// Make changes to the database here.
+		// Hint: Access models via this('modelName'), not an imported model to have
+		// access to the correct database connection. Otherwise Mongoose calls never return.
+		const amount = await userModel.find({ importHash: { $ne: null } }).countDocuments();
+		info(`${amount} user.importHashs will be removed`);
+		const limit = 200;
+		let looped = 0;
+		let removedImportHashes = 0;
+
+		while (looped < amount) {
+			const users = await userModel
+				.find({ importHash: { $ne: null } })
+				.sort({
+					updatedAt: 1,
+					createdAt: 1,
+				}).skip(looped)
+				.limit(limit)
+				.lean()
+				.exec();
+
+			for (const user of users) {
+				const accounts = await accountModel.find({ userId: user._id }).countDocuments();
```

Comment:
const $or =users.map((user) =>({userId: user._id}));
const accounts = await accountModel.find({ $or })
// check unique

Or
Promise.all(boundOfAccountRequests)
loop over all and test

Reply:
It is only nice to have for more performant migration. If you want to merge it without changes please discard my review statement. ;)